modules/todoList.js

```diff
@@ -0,0 +1,27 @@
+import resetIndex from './resetIndex.js';
+
+export default class TodoList {
+  constructor(list) {
+    this.list = list;
+  }
+
+  addItem(value) {
+    if (value) {
+      value = value.trim();
+      const ind = this.list.length + 1;
+      const listItem = {
+        description: value,
+        completed: false,
+        index: ind,
+      };
+      this.list.push(listItem);
+      localStorage.setItem('listData', JSON.stringify(this.list));
+    }
+  }
+
+  removeItem() {
+    this.list = this.list.filter((elem) => !elem.completed);
+    resetIndex(this.list);
+    return this.list;
+  }
+}
```

Comment:
Kindly consider commenting on all your code as you may come back to them later. It can also help other developers understand your code.